docs/topics/mtr-rn-new-features-1_1_0.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-1_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.1.0.
+
+.New reports
+{ProductShortName} generates rich reports using *PatternFly4*. The reports present the same content as in previous versions, but easier to navigate and presented better, in the consistent Web Console user interface. Users can still generate reports in the original format using the *legacyReports* advanced option.
+
+.New advanced options
+
+* *exiCodes* ensures that {ProductShortName} returns a zero return code with a successful analysis, and a non-zero return code with a failed analysis.
+
+* *exportSummary* generates a JSON file in the output folder of each analysed application. This file contains the story points and incident count (by category), along with the application's collection of technology tags.
```

Comment:
Please can we be consistent about our spelling of **analysed** and **analyzed** as we have used both variants in this doc.
**analyzed** is the preferred US spelling.